npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser/lockfile_parser.rb

```diff
@@ -96,8 +96,8 @@ def yarn_lock_dependencies
             parse_yarn_lock(yarn_lock).each do |req, details|
               next unless semver_version_for(details["version"])
               next if alias_package?(req)
-              next if workspace_package?(req)
-              next if req == "__metadata"
+              next if Experiments.enabled?(:yarn_berry) && workspace_package?(req)
+              next if Experiments.enabled?(:yarn_berry) && req == "__metadata"
```

Comment:
This has a side effect of no longer skipping over `workspace_packages` or `__metadata requirements` anymore unless the project has yarn_berry enabled. 

I don't think that's the intended effect though?

Reply:
This PR is targeting the yarn_berry branch these statements were added in that branch, so this just puts them behind a feature flag.